.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
not sure if the example is a clear and concise description. "I'm frustrated" does not sound like a beginning of a reasonable report. Perhaps something like "Ex. When trying to do X I need to Y because Z is missing"?